### PR TITLE
Add example of multimodal usage to pipeline tutorial

### PR DESCRIPTION
**Pull request by stevhliu**
As suggested by @NielsRogge, this PR adds a small example of how to use a multimodal pipeline (VQA) in the pipeline tutorial. 🙂

### CODE REVIEW

File: docs/source/en/pipeline_tutorial.mdx [hunk 1]
```diff
@@ -12,21 +12,21 @@ specific language governing permissions and limitations under the License.
...
 
-Take a look at the [`pipeline`] documentation for a complete list of supported tasks.
+Take a look at the [`pipeline`] documentation for a complete list of supported tasks and available parameters.
 
 </Tip>
 
 ## Pipeline usage
 
-While each task has an associated [`pipeline`], it is simpler to use the general [`pipeline`] abstraction which contains all the specific task pipelines. The [`pipeline`] automatically loads a default model and tokenizer capable of inference for your task. 
+While each task has an associated [`pipeline`], it is simpler to use the general [`pipeline`] abstraction which contains all the task specific pipelines. The [`pipeline`] automatically loads a default model and tokenizer capable of inference for your task.
```
Comment (sgugger) [inline docs/source/en/pipeline_tutorial.mdx:29]:
```suggestion
While each task has an associated [`pipeline`], it is simpler to use the general [`pipeline`] abstraction which contains all the task-specific pipelines. The [`pipeline`] automatically loads a default model and tokenizer capable of inference for your task. 
```

File: docs/source/en/pipeline_tutorial.mdx [hunk 2]
```diff
@@ -12,21 +12,21 @@ specific language governing permissions and limitations under the License.
 
 # Pipelines for inference
 
-The [`pipeline`] makes it simple to use any model from the [Model Hub](https://huggingface.co/models) for inference on a variety of tasks such as text generation, image segmentation and audio classification. Even if you don't have experience with a specific modality or understand the code powering the models, you can still use them with the [`pipeline`]! This tutorial will teach you to:
+The [`pipeline`] makes it simple to use any model from the [Hub](https://huggingface.co/models) for inference on tasks such as text generation, image segmentation, audio classification, and multimodal tasks like visual question answering. Even if you don't have experience with a specific modality or familiar with the underlying code behind the models, you can still use them for inference with the [`pipeline`]! This tutorial will teach you to:
```
Comment (sgugger) [inline docs/source/en/pipeline_tutorial.mdx:15]: Here the list makes it feel those are the only tasks supported (as it grows longer it feels like a list of examples but a comprehensive list). What about "on any tasks related to language, computer vision, speech, or even multimodal tasks."

File: docs/source/en/pipeline_tutorial.mdx [hunk 3]
```diff
@@ -146,3 +146,26 @@ Specify your vision task and pass your image to the classifier. The imaage can b
...
+
+## Multimodal pipeline
+
+The [`pipeline`] supports more than one modality. For example, a visual question answering (VQA) task combines text and image. Feel free to use any image link you like and a question you want to ask about the image. The image can be a URL or a local path to the image.
+
+For example, if you use the same image from the vision pipeline above:
+
+```py
+>>> image = "https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/pipeline-cat-chonk.jpeg"
+>>> question = "how chonky is this cat?"
```
Comment (sgugger) [inline docs/source/en/pipeline_tutorial.mdx:158]: chonky won't be super familiar for lot of non-native English speakers. I'd pick another question here.

**Review by sgugger — COMMENTED** [review]
Thanks for adding this, left a couple of comments.

File: docs/source/en/pipeline_tutorial.mdx (hunk 3 above)
Comment (stevhliu) [reply docs/source/en/pipeline_tutorial.mdx:158]: hehe you're right, i'll also not chonk-shame this cat 😁

**Review by sgugger — APPROVED** [review]

File: docs/source/en/pipeline_tutorial.mdx [hunk 4]
```diff
@@ -12,21 +12,21 @@ specific language governing permissions and limitations under the License.
 
 # Pipelines for inference
 
-The [`pipeline`] makes it simple to use any model from the [Model Hub](https://huggingface.co/models) for inference on a variety of tasks such as text generation, image segmentation and audio classification. Even if you don't have experience with a specific modality or understand the code powering the models, you can still use them with the [`pipeline`]! This tutorial will teach you to:
+The [`pipeline`] makes it simple to use any model from the [Hub](https://huggingface.co/models) for inference on any language, computer vision, speech, and multimodal tasks. Even if you don't have experience with a specific modality or familiar with the underlying code behind the models, you can still use them for inference with the [`pipeline`]! This tutorial will teach you to:
```
Comment (NielsRogge) [inline docs/source/en/pipeline_tutorial.mdx:15]:
```suggestion
The [`pipeline`] makes it simple to use any model from the [Hub](https://huggingface.co/models) for inference on any language, computer vision, speech, and multimodal tasks. Even if you don't have experience with a specific modality or aren't familiar with the underlying code behind the models, you can still use them for inference with the [`pipeline`]! This tutorial will teach you to:
```

File: docs/source/en/pipeline_tutorial.mdx [hunk 5]
```diff
@@ -12,21 +12,21 @@ specific language governing permissions and limitations under the License.
...
 
-Take a look at the [`pipeline`] documentation for a complete list of supported tasks.
+Take a look at the [`pipeline`] documentation for a complete list of supported tasks and available parameters.
 
 </Tip>
 
 ## Pipeline usage
 
-While each task has an associated [`pipeline`], it is simpler to use the general [`pipeline`] abstraction which contains all the specific task pipelines. The [`pipeline`] automatically loads a default model and tokenizer capable of inference for your task. 
+While each task has an associated [`pipeline`], it is simpler to use the general [`pipeline`] abstraction which contains all the task-specific pipelines. The [`pipeline`] automatically loads a default model and tokenizer capable of inference for your task.
```
Comment (NielsRogge) [inline docs/source/en/pipeline_tutorial.mdx:29]: Tokenizers are only relevant for NLP, not for vision, audio or multimodal tasks

File: docs/source/en/pipeline_tutorial.mdx [hunk 6]
```diff
@@ -67,7 +67,7 @@ Any additional parameters for your task can also be included in the [`pipeline`]
 
 ### Choose a model and tokenizer
 
-The [`pipeline`] accepts any model from the [Model Hub](https://huggingface.co/models). There are tags on the Model Hub that allow you to filter for a model you'd like to use for your task. Once you've picked an appropriate model, load it with the corresponding `AutoModelFor` and [`AutoTokenizer'] class. For example, load the [`AutoModelForCausalLM`] class for a causal language modeling task:
+The [`pipeline`] accepts any model from the [Hub](https://huggingface.co/models). There are tags on the Hub that allow you to filter for a model you'd like to use for your task. Once you've picked an appropriate model, load it with the corresponding `AutoModelFor` and [`AutoTokenizer`] class. For example, load the [`AutoModelForCausalLM`] class for a causal language modeling task:
```
Comment (NielsRogge) [inline docs/source/en/pipeline_tutorial.mdx:70]:
```suggestion
The [`pipeline`] accepts any model from the [Hub](https://huggingface.co/models). There are tags on the Hub that allow you to filter for a model you'd like to use for your task. Once you've picked an appropriate model, load it with the corresponding `AutoModel` and [`AutoTokenizer`] class. For example, load the [`AutoModelForCausalLM`] class for a causal language modeling task:
```

**Review by NielsRogge — APPROVED** [review]
Thanks for adding!

**Pull request merged**